docs/maps/connect-to-ems.asciidoc

```diff
@@ -0,0 +1,26 @@
+[role="xpack"]
+[[maps-connect-to-ems]]
+== Connecting to Elastic Maps Service
+
+The **Maps** application uses https://www.elastic.co/elastic-maps-service[Elastic Maps Service (EMS)]
+for tiles and vector shapes of administrative boundaries.
+
+The **Maps** application proxies requests for tile layer meta data, vector shapes meta data, and vector shapes
```

Comment:
The para needs an intro.  Something like:

Depending on your data, Maps proxies requests through either through the Kibana server or directly from the browser.  Requests for tile layer data, vector shapes, meta data, and vecto shapes go through the Kibana server.  Requests for EMS tiles are made directly from the browser to EMS.